app/controllers/admin/submitted_applications_reports_controller.rb

```diff
@@ -0,0 +1,17 @@
+module Admin
+  class SubmittedApplicationsReportsController < ApplicationController
+    before_action :authenticate_admin_user!
+    layout 'admin'.freeze
+
+    # GET /admin/submitted_applications_report
+    def show
+      @applications = submitted_applications
+    end
+
+    private
+
+    def submitted_applications
+      LegalAidApplication.submitted_applications
+    end
```

Comment:
I don't think we need to put this in a separate method when it's just calling a scope on a model.  So I would change line 8 to `@applications = LegalAidApplication.submitted_applications` and get rid of lines 10 onwards.